ADAL/src/request/ADAuthenticationRequest+Broker.m

```diff
@@ -205,6 +203,7 @@ + (ADAuthenticationResult *)processBrokerResponse:(NSURL *)response
     {
         protocolVersion = [msgVer integerValue];
     }
+    s_brokerProtocolVersion = msgVer;
```

Comment:
I am also thinking the same :) 
But I am having trouble to inject the broker telemetry event there. When broker returns back, ADBrokerHelper calls the class method of ADAuthenticationRequest to handle the response from broker.